common/app/common/configuration.scala

```diff
@@ -474,6 +474,21 @@ class GuardianConfiguration extends Logging {
     lazy val config = configuration.getMandatoryStringProperty("front.config")
   }
 
+  object targeting {
+    lazy val campaignsUrl = configuration.getStringProperty("targeting.campaignsUrl")
+
+    // Total number of rules allowed per campaign, any campaigns with more than one rule will be filtered
+    lazy val ruleLimit: Int =
+      Try(configuration.getStringProperty("targeting.ruleLimit").get.toInt)
```

Comment:
I figured if you wanted to change them it would be easier to override them as config instead of going through a code change hence I implemented them as config with a reasonable default